packages/react/src/themes/teams/components/Icon/svg/icons/index.ts

```diff
@@ -125,6 +126,7 @@ export default {
   'call-video-off': callVideoOff,
   'call-control-present-new': callControlPresentNew,
   'call-control-stop-presenting-new': callControlStopPresentingNew,
+  'call-missed-line-filled': callMissedLineFilled,
```

Comment:
I would avoid the name filled, as the difference whether the icon will be filled or not is dependent on a prop (outline)